publish/views/asset.py

```diff
@@ -34,11 +35,9 @@ class Meta(AssetSerializer.Meta):
 
 
 class AssetFilter(filters.FilterSet):
-    path_prefix = filters.CharFilter(lookup_expr='startswith', field_name='path')
-
     class Meta:
         model = Asset
-        fields = ['path_prefix']
```

Comment:
@AlmightyYakob One final question: the removal of `path_prefix` as a general filter doesn't affect the fact that `path_prefix` is used as a query string within the `paths` action?

Reply:
No, it's just a coincidence, and part of the reason why I wanted to replace the `path_prefix` filter.